ui/Makefile

```diff
@@ -54,22 +56,34 @@ GOBINDATA_SOURCES       := $(BOWER_COMPONENTS)/d3/d3.min.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.js \
+                           $(BOWER_COMPONENTS)/moment/min/moment.min.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.min.js \
                            $(GOBINDATA_FONTS)
 GOBINDATA_DEBUG_SOURCES := $(BOWER_COMPONENTS)/d3/d3.js \
                            $(BOWER_COMPONENTS)/lodash/lodash.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.js \
+                           $(BOWER_COMPONENTS)/moment/moment.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.js \
```

Comment:
So this includes a copy of the zoneinfo database? We should really try to avoid that. It's not as critical if it's just in the UI, but it would still suck for people to have to update their database servers just to get correct timezone info (which changes every couple of months. On the server side this data lives outside the cockroach binary so it's easier to udpate). Since, as discussed offline, we're going to make a special-purpose endpoint for queries from the UI, we should try to do all time-zone-related operations on the server side using the system zoneinfo DB.